plugins/org.jkiss.dbeaver.model.ai/src/org/jkiss/dbeaver/model/ai/AIMessage.java

```diff
@@ -39,11 +42,13 @@ public class AIMessage {
      * Creates AI message
      */
     public AIMessage(
+        @NotNull UUID uuid,
```

Comment:
IT conflicts with https://github.com/dbeaver/pro/issues/5835
We must sync.
In this ticket we didn't plan to change any model.